docs/howtos/qosserver.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: QoS Server
```

Comment:
Should we have a more descriptive title? QoS is an MPS thing, I think "Measuring latency" can be more descriptive